tests/tests_suite_test.go

```diff
@@ -52,9 +52,10 @@ var (
 )
 
 const (
-	amd64 = "amd64"
-	s390x = "s390x"
-	arm   = "aarch64"
+	amd64   = "amd64"
+	s390x   = "s390x"
+	arm     = "arm"
```

Comment:
will this be arm64 ?
oc get nodes -o json | jq '.items[].status.nodeInfo.architecture'
"arm64"